requirements.txt

```diff
@@ -1,6 +1,3 @@
-cython>=3.0.0
-numpy
+numpy>=1.10.0
 greenlet>=3.0.0
-
-# Required for the charm++ build:
-cmake 
+cython>=3.0.0
```

Comment:
Just add a linebreak after the last line here

Reply:
done